Chat/ClientApp/src/core/sideEffects.ts

```diff
@@ -302,10 +376,14 @@ const removeThreadMemberByUserId = (userId: string) => async (dispatch: Dispatch
     return;
   }
   let chatThreadClient = await chatClient.getChatThreadClient(threadId);
-  let response = await chatThreadClient.removeMember({
-    communicationUserId: userId
-  });
-  if (response._response.status === TOO_MANY_REQUESTS_STATUS_CODE) {
+  try {
+    let response = await chatThreadClient.removeParticipant({
+      communicationUserId: userId
+    });
+  }
+  catch(error) {
+    // TOO_MANY_REQUESTS_STATUS_CODE
+    console.log(error);
```

Comment:
can we test this?

Reply:
I don't think this is going to be an issue anymore but we should still bug bash this with the team